mexca/data.py

```diff
@@ -479,6 +479,7 @@ class TranscriptionData:
     index: int
     text: str
     speaker: Optional[str] = None
+    confidence: float # probability of transcription accuracy
```

Comment:
```suggestion
    confidence: Optional[float] = None # probability of transcription accuracy
```

 Python cannot handle non-default arguments after default ones, so this should fix a couple of issues in the actions.

Reply:
Ah I see... I think I looked in the wrong place in the log for the error about 'word_timestamps'... I will make this change and see if it solves the issue. But isn't it strange that the 'word_timestamps' error shows up at all?

Reply:
For which action do you see that error?